integration-tests/oidc-tenancy/src/main/java/io/quarkus/it/keycloak/TenantResource.java

```diff
@@ -40,7 +42,9 @@ public String userNameService(@PathParam("tenant") String tenant) {
         String name = getNameServiceType();
         if ("tenant-d".equals(tenant) || "tenant-b-no-discovery".equals(tenant)) {
             UserInfo userInfo = getUserInfo();
-            name = name + "." + userInfo.getString("preferred_username");
+            if (userInfo.getPropertyNames().contains(PREFERRED_USERNAME)) {
```

Comment:
Instead of returning the property names via a getter, isn't better to introduce a `UserInfo.contains(String)` instead?

Reply:
@gastaldi the users just wanted to see the whole set of property names as he was not sure what is in that UserInfo, here I just added this awkward check to test the new method :-), but yeah, let me add a shortcut as well, make sense

Reply:
@gastaldi Just updated it, kept the `getPropertyNames` test check but also added `contains` check

Reply:
Looks good, thanks!